lib/foreman_theme_satellite/documentation.rb

```diff
@@ -5,8 +5,8 @@ module ForemanThemeSatellite
   module Documentation
     USER_GUIDE_DICTIONARY = {
       # Ansible
-      'ImportingRoles' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Importing_Ansible_Roles_and_Variables_ansible",
-      'Variables' => "#{ForemanThemeSatellite.documentation_root}/managing_configurations_using_ansible_integration_in_red_hat_satellite/getting_started_with_ansible_in_satellite_ansible#Overriding_Ansible_Variables_in_satellite_ansible",
```

Comment:
I'm not sure if it's a good idea to replace these keys. It's possible that they are coded elsewhere as identifiers and if we remove them, then we lose the connection. But I'll try to find someone who really understands it to ack this.

Reply:
It was correct to remove them, because they were removed from the source code.

@girijaasoni please remove the lines again.